steps/infra/aws/main.tf

```diff
@@ -41,6 +41,13 @@ module "masters" {
   user_data_igns      = "${var.tectonic_ignition_masters}"
 }
 
+module "iam" {
+  source = "../../../modules/aws/iam"
+
+  cluster_name     = "${var.tectonic_cluster_name}"
+  worker_iam_role  = "${var.tectonic_aws_worker_iam_role_name}"
```

Comment:
There's an extra space here which `terraform fmt` wants to remove.  I'll file a fixup and try and figure out why our CI tests missed it.

Reply:
> I'll file a fixup...

Filed as #269.